aten/src/ATen/core/op_registration/op_registration.cpp

```diff
@@ -26,6 +26,15 @@ class RegisterOperators::OperatorRegistrar final {
     }
   }
 
+  explicit OperatorRegistrar(FunctionSchema&& schema, KernelFunctionWrapper* wrapper)
```

Comment:
If you're changing the wrapper to live on OperatorOptions instead of on RegisterOperators (see my comment below), then you probably don't need this separate constructor anymore and can just do it in the already existing constructor above.